app/src/main/java/com/example/ptvproject/ui/selecttrainline/SelectTrainLineScreen.kt

```diff
@@ -1,39 +1,34 @@
 package com.example.ptvproject.ui.selecttrainline
 
 
-import androidx.compose.foundation.BorderStroke
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.background
 import androidx.compose.foundation.border
 import androidx.compose.foundation.layout.*
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
 import androidx.compose.foundation.selection.selectable
-import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material.*
-import androidx.compose.material.SnackbarDefaults.backgroundColor
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.ArrowBack
 import androidx.compose.runtime.*
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.lifecycle.viewmodel.compose.viewModel
 import com.example.ptvproject.R
 import com.example.ptvproject.ui.theme.PTVprojectTheme
-import java.time.ZonedDateTime
 import androidx.compose.foundation.layout.Column as Column
-import com.example.ptvproject.ui.selecttrainline.ConfirmTrainLineDialog as ConfirmTrainLineDialog
 
 @Composable
 fun SelectTrainLineScreen(
     selectTrainLineViewModel: SelectTrainLineViewModel = viewModel(),
-    //onTrainLineSelected: (departureTime: ZonedDateTime, stationName: String, trainLine: String, direction: String) -> Unit
-) {
+    onTrainLineSelected: (String, String, String, String) -> Unit,
```

Comment:
it would be great to see the parameter names for each of these types, like the older code showed!